G1ANT.Addon.UI/Commands/UIGetCheckboxStateCommand.cs

```diff
@@ -0,0 +1,33 @@
+﻿using G1ANT.Addon.UI.Api;
+using G1ANT.Addon.UI.Structures;
+using G1ANT.Language;
+
+namespace G1ANT.Addon.UI.Commands
+{
+    [Command(Name = "ui.getcheckboxstate", Tooltip = "This command returns targeted checkbox's state")]
+    public class UIGetCheckboxStateCommand : Command
```

Comment:
for me this command is not consistent, command name and tooltip suggest that command is for Checkbox control only, I would like to split code to separate command to check Radio Button control but I know that you make this exactly what was written in Task, consult this with Przemek Witusik (he was a reviewer previously :))

Reply:
I agree. I will verify with @pwitusik  before merging

Reply:
OK, I agree name of the command is confusing. I don't want to split the same functionality to different commands, I prefer to change the name of the command to be common for checkboxes and radio buttons. Any suggestions for the different name?